spec/GameSpec.js

```diff
@@ -18,6 +22,14 @@ describe("Game", function() {
     expect(status).toEqual("in progress");
   });
 
+  it("can be initialized with a client", function() {
+    expect(game.get('client')).toBeDefined();
+  });
+
+  it("can be initialized with a compiler", function() {
+    expect(game.get('compiler')).toBeDefined();
+  });
+
```

Comment:
Probably not... they were just the first tests I wrote to make sure nothing was broken.